packages/core/src/Dialog/Dialog.tsx

```diff
@@ -19,6 +20,7 @@ export type DialogProps = Omit<OverflowProps, 'overflow'> & {
   fullWidth?: boolean
   headerColorScheme?: keyof ColorSchemes
   headerContent?: string | React.ReactNode
+  headerContentProps?: GridProps & SpaceProps
```

Comment:
If you remove `SpaceProps` does it still work? I believe GridProps already supports SpaceProps from one of the types it unions with

Reply:
It does! I'll remove the import.